tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -0,0 +1,200 @@
+
+
+*** Keywords ***
+Open Data Science Projects Home Page
+    [Documentation]    Verifies submenu Settings > Cluster settings" is visible
+    Page Should Contain    Data Science Projects
+    Click Link      Data Science Projects
+    Wait Until Page Contains    View your existing projects or create new projects.    timeout=30
+    Wait Until Page Contains Element    ${DS_PROJECT_XP}    timeout=30
+    Maybe Wait For Dashboard Loading Spinner Page
+
+Is Data Science Projects Page Open
+    Close Generic Modal If Present
+    ${page_open}=   Run Keyword And Return Status   Page Should Contain Element     ${DS_PROJECT_XP}
+    [Return]    ${page_open}
+
+Open Data Science Project Details Page
+    [Documentation]    Verifies submenu Settings > Cluster settings" is visible
+    [Arguments]     ${project_title}
+    ${is_open}=    Is Data Science Projects Page Open
+    IF    ${is_open} == ${FALSE}
+        Open Data Science Projects Home Page
+    END
+    Click Link    ${project_title}
+    Wait Until Project Is Open    project_title=${project_title}
+    Maybe Wait For Dashboard Loading Spinner Page
+
+Delete Data Science Project
+    [Arguments]     ${project_title}    ${press_cancel}=${FALSE}
+    ${is_open}=    Is Data Science Projects Page Open
+    IF    ${is_open} == ${FALSE}
+        Open Data Science Projects Home Page
+    END
+    Project Should Be Listed    project_title=${project_title}
+    Click Action From Actions Menu    item_title=${project_title}    item_type=project   action=Delete
+    Handle Deletion Confirmation Modal    item_title=${project_title}    item_type=project    press_cancel=${press_cancel}
+    ...          additional_msg=It will destroy all workbenches, storages, data connections and other resources in ${project_title}
+
+Wait Until Project Is Open
+    [Arguments]     ${project_title}
+    Wait Until Page Contains Element    xpath=//h1[contains(text(),"${project_title}")]    timeout=30
+    Maybe Wait For Dashboard Loading Spinner Page
+
+Project Should Be Listed
+    [Arguments]     ${project_title}
+    Run keyword And Continue On Failure     Page Should Contain Element    xpath=//td/a[text()="${project_title}"]
+
+Project's Owner Should Be
+    [Arguments]     ${project_title}    ${expected_username}
+    Run keyword And Continue On Failure     Page Should Contain Element    xpath=//td[a[text()="${project_title}"]]/small[text()="${expected_username}"]
+
+Create Data Science Project
+    [Arguments]    ${title}    ${description}    ${resource_name}=${NONE}
+    Click Button    ${PROJECT_CREATE_BTN_XP}
+    Wait Until Page Contains Element    ${TITLE_INPUT_XP}
+    Run Keyword And Warn On Failure     Element Should Be Disabled    ${GENERIC_CREATE_BTN_XP}
+    Input Text    ${TITLE_INPUT_XP}    ${title}
+    Input Text    ${DESCR_INPUT_XP}    ${description}
+    Run Keyword And Continue On Failure    Validate Generated Resource Name    project_title=${title}
+    IF    "${resource_name}" == "${NONE}"
```

Comment:
shouldn't this be `IF    "${resource_name}" != "${NONE}"`?

Reply:
yes! thank you